### DISCUSSION TITLE
Using Xelatex in place of pdflatex

### DISCUSSION THREAD

**Discussion by mauzil** [doxygen/doxygen#11278] [Q&A]
Hi all.

Is it possible to use xelatex in place of pdflatex, setting xelatex direcly in Doxyfile?
For some strange reason in my Artix installation, pdflatex crashes due to poor resources.

pdflatex tell me this

```
 TeX capacity exceeded, sorry [pool size=5777463].
<to be read again> 
                   \baselineskip 
l.1352 ...onfig(ipv4,\ service-\/>nameservers());}
                                                  
If you really absolutely need more capacity,
you can ask a wizard to enlarge me.

 
Here is how much of TeX's memory you used:
 185630 strings out of 475469
 5777463 string characters out of 5777463
 3007245 words of memory out of 5000000
 69262 multiletter control sequences out of 15000+600000
 615690 words of font info for 115 fonts, out of 8000000 for 9000
 350 hyphenation exceptions out of 8191
 101i,16n,117p,2122b,7901s stack positions out of 10000i,1000n,20000p,200000b,200000s
!  ==> Fatal error occurred, no output PDF file produced!
```

While with xelatex all works fine

**Comment by albert-github** [comment]
It should be possible to use xelatex instead of pdflatex.
- Please specify the **full** doxygen version used (`doxygen -v`).
 
Not sure this is directly a doxygen problem seen the message:
```
TeX capacity exceeded, sorry
```
so probably some things have to be enlarged in your latex settings (outside the scope of doxygen).

- Can you please attach a, small, self contained example (source+configuration file in a, compressed, tar or zip file!) that allows us to reproduce the problem? Please don't add external links as they might not be persistent (also references to GitHub repositories are considered non persistent).
- Please also specify the **full** doxygen version used (`doxygen -v`).
